tuna/miopen/driver/base.py

```diff
@@ -200,7 +205,7 @@ def compose_input_t(self) -> Dict[str, int]:
 
     return i_dict
 
-  def decompose_input_t(self, db_obj) -> bool:
+  def decompose_input_t(self, db_obj: ConvolutionConfig) -> bool:
```

Comment:
do not impose subclass, leave as Any

Reply:
This issue is that base may service additional subclasses, of which the two that are currently implemented may not be all inclusive. Additionally it is not the role of the base class to record all of its subclasses.

Reply:
Ok, Rolled back to Any type